modules/serverless-tls-creating-cluster-issuer-eventing.adoc

```diff
@@ -0,0 +1,45 @@
+// Module included in the following assemblies:
+//
+// * /serverless/Eventing/serverless-config-tls-encryption-eventing.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="serverless-tls-creating-cluster-issuer-eventing_{context}"]
+= Creating a `ClusterIssuer` resource for Eventing
+
+`ClusterIssuers` are Kubernetes resources that represent certificate authorities (CAs) capable of generating signed certificates by honoring certificate signing requests.
+
+.Prerequisites
+
+* You have cluster administrator permissions on {ocp-product-title}, or you have cluster or dedicated administrator permissions on {rosa-product-title} or {dedicated-product-title}.
+* You have installed the {ServerlessOperatorName}.
+* You have installed the {cert-manager-operator}.
+* You have installed the OpenShift (`oc`) CLI.
+
+.Procedure
+
+. Create the `knative-eventing-ca-issuer` `ClusterIssuer` resource as follows:
++
+[source,yaml]
+----
+# This is the issuer that every Eventing component should use to issue their server's certs.
+apiVersion: cert-manager.io/v1
+kind: ClusterIssuer
+metadata:
+  name: knative-eventing-ca-issuer
+spec:
+  ca:
+    secretName: knative-eventing-ca <1>
+----
```

Comment:
Somehow when I click on `copy code` for this snipped I get (note the missing `#` at the start)

```
This is the issuer that every Eventing component should use to issue their server's certs.
apiVersion: cert-manager.io/v1
kind: ClusterIssuer
metadata:
  name: knative-eventing-ca-issuer
spec:
  ca:
    secretName: knative-eventing-ca 
```

is it the same for you?

Reply:
If this is bug somewhere in the website builder, while that gets fixed, we could remove that YAML comment there and add it as part of the point with a text like "Create the `knative-eventing-ca-issuer` `ClusterIssuer` resource that every Eventing component will use to issue their server's certs as follow:"

Reply:
@pierDipi I checked it from my end and I am getting the same thing. So I will add your suggestion in the content. I will remove the YAML comment.